CONTRIBUTING.md

```diff
@@ -128,6 +128,20 @@ make server
 Any time changes are made to the Go code, you will have to stop the server and run the above command again.
 Once the Meshery server is up and running, you should be able to access Meshery on your `localhost` on port `9081` at `http://localhost:9081`.
 
+**Please note**: If you get error while starting the server as **"Meshery Development Incompatible"** then follow the below guideline 👇
+
+Potential Solution: 
+
+-  Go to your meshery folder in your local-system where you’ve cloned it.
+Execute: `git remote add upstream https://github.com/meshery/meshery
+git fetch upstream`
```

Comment:
```suggestion
Execute: 

```
git remote add upstream https://github.com/meshery/meshery
git fetch upstream
```

Reply:
<img width="682" alt="Screenshot 2023-06-28 at 1 10 59 AM" src="https://github.com/meshery/meshery/assets/48255244/c12250e1-05c9-49f7-b06c-9708c6e1b14e">

The markdown reader messes it up, but this is the change